source/_docs/composer-drupal-8.md

```diff
@@ -9,6 +9,16 @@ contributors:
 
 [Composer](https://getcomposer.org/) is a dependency management tool that allows PHP code to be more easily shared across projects. As Drupal 8 was written, many of its subsystems written specifically for Drupal were replaced by more widely used packages. For instance, Drupal's notoriously unwieldy function for making external requests, `drupal_http_request()`, [was replaced](https://www.drupal.org/node/1862446) by [Guzzle](http://guzzlephp.org/). Guzzle is included in Drupal 8 via Composer.
 
+Before using Composer to manage your site development you should choose a workflow. By default, Pantheon controls and distributes core updates via git commits, which conflicts with updates processed by Composer. We suggest developers using Composer install their sites with the [Example Drops 8 Composer](https://github.com/pantheon-systems/example-drops-8-composer), which moves Drupal to a "nested docroot". The repository README file covers the two common methods of deployment.
+
+<div class="alert alert-info">
+<h4 class="info">Note</h4><p markdown="1">
+The methods described by the Example Drops 8 Composer repository replace the standard Drupal 8 upstream. By choosing this method, you become responsible for ensuring that your core remains up to date, per our [scope of support](/docs/getting-support/#scope-of-support)
+</p>
+</div>
+
+If you'd instead prefer to the default Drupal 8 installation provided by Pantheon, use a tool like the [Composer Merge Plugin](https://github.com/wikimedia/composer-merge-plugin), which parses multiple `composer.json` files into a single version when running updates. This method is the most likely to see merge conflicts, as core updates will also update Drupal's `composer.json`.
```

Comment:
Drupal provides the Composer Merge Plugin, and uses it automatically; this isn't something that the user has to choose to do. Also, the composer merge plugin isn't really the thing that is causing conflicts in this workflow. Rather, the fact that both  Composer and the Pantheon upstream may change the same files in the vendor directory, and both will update the composer.lock file.